jax/_src/numpy/ufuncs.py

```diff
@@ -253,9 +253,54 @@ def bitwise_count(x: ArrayLike, /) -> Array:
   # Following numpy we take the absolute value and return uint8.
   return lax.population_count(abs(x)).astype('uint8')
 
-@implements(np.right_shift, module='numpy')
 @partial(jit, inline=True)
 def right_shift(x1: ArrayLike, x2: ArrayLike, /) -> Array:
+  r"""Right shift the bits of ``x1`` to the amount specified in ``x2``, element by
```

Comment:
The first line of a docstring should be a concise summary of the function, followed by one blank line. If you want extra description below the first line, put it in a separate paragraph.

Reply:
I'd remove "element by element" to make it fit on one line.